packages/backend-function/src/layer_parser.test.ts

```diff
@@ -28,6 +28,8 @@ void describe('AmplifyFunctionFactory - Layers', () => {
   let rootStack: Stack;
   let getInstanceProps: ConstructFactoryGetInstanceProps;
   let resourceNameValidator: ResourceNameValidator;
+  // mock lazy to return same region
+  mock.method(Lazy, 'string', () => 'us-east-1');
```

Comment:
Is it possible to mock `stack.region` instead?
Mocking `Lazy.string` might become problematic if we get more of that in unrelated places.

Reply:
I'll see if I can mock the region directly

Reply:
I set the region for the app in the unit test and updated the logic for getting stack region in [79c37d8](https://github.com/aws-amplify/amplify-backend/pull/2188/commits/79c37d8142bd2c1aa8b73b8d14a20f5ab391e8b6)